openmetadata-ui/src/main/resources/ui/src/pages/APICollectionPage/APICollectionVersionPage.tsx

```diff
@@ -228,13 +231,13 @@ const APICollectionVersionPage = () => {
       if (cursorType) {
         getAPICollectionEndpoints({
           paging: {
-            [cursorType]: apiEndpoints.paging[cursorType],
+            [cursorType]: paging[cursorType],
           },
         });
       }
       setCurrentEndpointsPage(currentPage);
     },
-    [apiEndpoints, getAPICollectionEndpoints]
+    [paging, getAPICollectionEndpoints, handlePageChange]
```

Comment:
`handlePageChange` is not used in the function

Reply:
Removed the dependency.